packages/by-name/microsoft/genpolicy/package.nix

```
@@ -71,6 +71,10 @@ rustPlatform.buildRustPackage rec {
       # This avoids printing the entire annotation on log level debug, which resulted in errors of the logtranslator.go
       # TODO(jmxnzo): remove when https://github.com/kata-containers/kata-containers/pull/10647 is picked up by microsoft/kata-containers fork
       ./0009-genpolicy-do-not-log-policy-annotation-in-debug.patch
+      # Patches the RootfsPropagation check in allow_create_container_input to allow setting up volumes. It reflects
+      # 0012-genpolicy-support-mount-propagation-and-ro-mounts.patch on upstream kata repo, but drops the patched propagation mode
```

```suggestion
      # genpolicy-support-mount-propagation-and-ro-mounts.patch on kata.genpolicy, but drops the patched propagation mode
```
patch numbers change over time.

---

The second part of the suggestion was not applied, but should be.